processors/config/anaSimps_2016_cfg.py

```diff
@@ -51,22 +56,16 @@
 vtxana.parameters["isRadPDG"] = options.isRadPDG
 vtxana.parameters["makeFlatTuple"] = options.makeFlatTuple
 vtxana.parameters["beamPosCfg"] = ""
-vtxana.parameters["pSmearingFile"] =  os.environ['HPSTR_BASE']+"/utils/data/smearingFile_2016_all_12112023.root"
-if options.isData:
+if options.isData and options.year == 2016:
     vtxana.parameters["v0ProjectionFitsCfg"] = os.environ['HPSTR_BASE']+'/analysis/data/v0_projection_2016_config.json'
-else:
-    vtxana.parameters["v0ProjectionFitsCfg"] = os.environ['HPSTR_BASE']+'/analysis/data/v0_projection_2016_mc_7800_config.json'  #For tritrig and wab mc
-    #vtxana.parameters["v0ProjectionFitsCfg"] = os.environ['HPSTR_BASE']+'/analysis/data/v0_projection_2016_mc_signal_config.json' #For signal (accidentally gen with bspt=(0,0)
-
-if options.isData:
-    vtxana.parameters["eleTrackTimeBias"] = -1.5
-    vtxana.parameters["posTrackTimeBias"] = -1.5
-else:
-    vtxana.parameters["eleTrackTimeBias"] = -2.2 #MC
-    vtxana.parameters["posTrackTimeBias"] = -2.2 #MC
-    #vtxana.parameters["eleTrackTimeBias"] = -5.5 #MC For TTs new smearing samples...due to readout bug
-    #vtxana.parameters["posTrackTimeBias"] = -5.5 #MC For TTs new smearing samples...due to readout bug
-
+    vtxana.parameters["trackBiasCfg"] = os.environ['HPSTR_BASE']+'/utils/data/track_bias_corrections_data_2016.json'
+elif not options.isData and options.year == 2016:
+    print('Running MC')
+    vtxana.parameters["trackBiasCfg"] = os.environ['HPSTR_BASE']+'/utils/data/track_bias_corrections_tritrig_2016.json'
```

Comment:
The JSON files referenced here are not in hpstr.

Reply:
You do add `analysis/data/2016_mc_ana_corrections.json`, is that a copy or are they supposed to be different? Are you treating "ana corrections" and "bias corrections" the same?